### PR TITLE
chore: output targeting enable disable as a tree

### PR DESCRIPTION
**Pull request by jsalaber**
# Changes

- Refactor enable / disable targeting commands to use common method
- output response of enable / disable targeting as a tree

### CODE REVIEW

**Comment by jsalaber** [comment]
Current dependencies on/for this PR:
* main
  * **PR #211** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/cli/211" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/DevCycleHQ/cli/211?utm_source=stack-comment).

File: src/utils/targeting/index.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { disableTargeting, enableTargeting } from '../../api/targeting'
+import inquirer from '../../ui/autocomplete'
+import { featurePrompt, EnvironmentPromptResult, environmentPrompt } from '../../ui/prompts'
+
+export const getResponsesForEnableDisableTargeting = async (
```
Comment (laurawarr) [inline src/utils/targeting/index.ts:5]: Would this be more accurate as something like `getFeatureAndEnvironmentArgs`? I had to read the function code to understand what "responses" this is getting. I could also see us using this for targeting commands other than enable/disable

File: src/commands/targeting/enable.ts [hunk 2]
```diff
@@ -23,47 +28,26 @@ export default class EnableTargeting extends Base {
...
-            responses.featureKey,
-            responses.environmentKey
-        )
-        return this.writer.showResults(enableTargetingForFeatureAndEnvironment)
+            responses.featureKey as string,
+            responses.environmentKey as string
+        ) 
+    
+        if (flags.headless) {
+            this.writer.showResults(getResponsesForEnableDisableTargeting)
```
Comment (laurawarr) [inline src/commands/targeting/enable.ts:45]: shouldn't this take `updatedTargeting` rather than `getResponsesForEnableDisableTargeting`?

Comment (jsalaber) [reply src/commands/targeting/enable.ts:45]: oops yeah

File: src/commands/targeting/disable.ts [hunk 3]
```diff
@@ -23,47 +25,26 @@ export default class DisableTargeting extends Base {
...
-        return this.writer.showResults(disableTargetingForFeatureAndEnvironment)
+            responses.featureKey as string,
+            responses.environmentKey as string
+        ) 
+    
+        if (flags.headless) {
+            this.writer.showResults(updatedTargeting)
+        } else {
+            // TODO: reuse the data fetched for the prompts
+            const environments = await fetchEnvironments(this.authToken, this.projectKey)
```
Comment (laurawarr) [inline src/commands/targeting/disable.ts:45]: we could just fetch the related environment by key, since we're only updating one

**Review by laurawarr — APPROVED** [review]
one small comment regarding fetch the env, but otherwise looks good

**Pull request merged**